src/Command/InfectionCommand.php

```diff
@@ -148,7 +148,7 @@ protected function configure()
                 'log-verbosity',
                 null,
                 InputOption::VALUE_OPTIONAL,
-                'Log verbosity level. 1 - full logs format, 2 - short logs format.',
+                'Log verbosity level. 1 - full logs format, 2 - short logs format, 3 - no logs.',
```

Comment:
usually it's the opposite though: the higher the level the more complete the logs are

Reply:
I know, but now it is this way. So i'd propose to keep it how it is right now, but add a notice to the end users that it will be changed.

And then change it in the next release.